dbt/adapters/snowflake/relation_configs/dynamic_table.py

```diff
@@ -21,6 +35,147 @@ class SnowflakeDynamicTableConfig(RelationConfigBase):
     """
 
     name: str
+    schema_name: str
+    database_name: str
     query: str
     target_lag: SnowflakeDynamicTableTargetLagConfig
     warehouse: str
+
+    @property
+    def path(self) -> str:
```

Comment:
Howabout relation path?

Reply:
Since this will always be used in the form `dynamic_table.path`, I'm not keen on `dynamic_table.relation_path` since it feels like doublespeak. I chose `path` because I believe we call it that in `dbt-core` somewhere. However, `path` is generic; what this really is is the "fully qualified path". And since we're in `relation_config` land, we should be talking in database parlance. I'll update it to `fully_qualified_path`, but let me know your thoughts on that name.

Reply:
I _love_ `fully_qualified_path`.